project4/index.html

```diff
@@ -0,0 +1,232 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Maven+Pro:400,500,700,900&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+
+<header>
+
+    <nav>
+        <ul>
```

Comment:
Since you float a child element of this list, you must apply the clearfix to clear your floats.
```suggestion
        <ul class="clearfix">
```
I'll send you a reminder of the clearfix CSS via Slack.